source/Character.ts

```diff
@@ -1277,21 +1277,19 @@ export class Character extends Observer implements CharacterData {
                 if (typeof data == "object") {
                     if (data.response == "bank_restrictions" && data.place == "compound") {
                         this.socket.off("game_response", gameResponseCheck)
-                        reject("You can't compound items in the bank.")
+                        reject(new Error("You can't compound items in the bank."))
                     } else if (data.response == "compound_chance" && data.item.name == item1Info.name && data.item.level == item1Info.level) {
                         resolve(data)
                     }
                 } else if (typeof data == "string") {
-                    if (data == "compound_no_item") {
-                        this.socket.off("game_response", gameResponseCheck)
-                        reject()
-                    }
+                    this.socket.off("game_response", gameResponseCheck)
+                    reject(new Error(data))
```

Comment:
This log message no longer exists in the game files, seems like the validation was moved into the client.